internal/engine/interpreter/interpreter_test.go

```diff
@@ -117,31 +123,49 @@ func TestInterpreter_Engine_NewModuleEngine_InitTable(t *testing.T) {
 func TestInterpreter_ModuleEngine_Call(t *testing.T) {
 	defer functionLog.Reset()
 	enginetest.RunTestModuleEngine_Call(t, et)
-	require.Equal(t, `--> .$0(1,2)
+	require.Equal(t, `
+--> .$0(1,2)
 <-- (1,2)
-`, functionLog.String())
+`, "\n"+functionLog.String())
 }
 
 func TestInterpreter_ModuleEngine_Call_HostFn(t *testing.T) {
 	defer functionLog.Reset()
 	enginetest.RunTestModuleEngine_Call_HostFn(t, et)
-	require.Equal(t, `==> .$0(3)
-<== (3)
---> imported.wasm_div_by(1)
+	require.Equal(t, `
+--> imported.div_by.wasm(1)
+<-- (1)
+--> host.div_by.wasm(1)
+<-- (1)
+--> imported.call->div_by.go(1)
+	--> host.div_by.wasm(1)
+	<-- (1)
+<-- (1)
+--> importing.call_import->call->div_by.go(1)
+	--> imported.call->div_by.go(1)
+		--> host.div_by.wasm(1)
+		<-- (1)
+	<-- (1)
 <-- (1)
-==> host.host_div_by(1)
+--> host.read_mem.wasm()
```

Comment:
this is log evidence that reading memory can work (though the unit tests also don't fail)

"==>" implies a go-defined function call and "-->" a wasm-defined one

Reply:
ps I'm ok removing log checking from this test, as there are enough tests out there that we don't really need to check the whole thing.